yesod-auth/Yesod/Auth/Util/PasswordStore.hs

```diff
@@ -71,7 +66,7 @@
 -- iteration count. This does not have a significant effect on security, but can
 -- be handy for compatibility with other code.
 
-module Yesod.PasswordStore (
+module Yesod.Auth.Util.PasswordStore (
```

Comment:
Can you add `@since 1.4.18` comments to the module and all identifiers exposed here?

Reply:
Of course!  Just pushed../